src/parcsr_ls/par_mgr_setup.c

```diff
@@ -1830,16 +1830,15 @@ hypre_MGRSetup( void               *mgr_vdata,
       TODO (VPM): move this block closer to global smoother setup */
    for (j = 0; j < num_c_levels; j++)
    {
-      if (((mgr_data -> num_relax_sweeps)[j] > 0) && (l1_norms[j] == NULL))
+      if (!l1_norms[j])
```

Comment:
Why is the check for num_relax_sweeps > 0 removed?

Reply:
Because we need to compute l1_norms since global relaxation might need it. 

To be more precise, there are situations where l1_norms is not required. So, I could have been more strict here by adding checks instead of removing the one we have.

I plan to improve this in a future PR